### PR TITLE
Implement label and description for the datepicker component

### PR DESCRIPTION
**Pull request by JammingBen**
* Remove placeholders
* Add required label to datepickers
* Add optional description message to datepickers
* Add aria-describedby which references the description
* Add aria-invalid property (which is basically always false as the datepicker does not allow invalid inputs -> no errors)

### CODE REVIEW

File: src/components/OcDatepicker.vue [hunk 1]
```diff
@@ -50,20 +63,18 @@ export default {
    * type(s). See examples below:
    */
   props: {
+    id: {
```
Comment (JanAckermann) [inline src/components/OcDatepicker.vue:66]: replace `datepickerId` with `id` :+1

File: src/components/OcDatepicker.vue [hunk 2]
```diff
@@ -1,24 +1,37 @@
...
-    :type="type"
-    :phrases="phrases"
-    :min-datetime="minDatetime"
-    :max-datetime="maxDatetime"
-    :placeholder="placeholder"
-    @cancel="$_cancel()"
-    @input="$_input()"
-  ></datetime>
+  <div>
+    <label class="oc-label" :for="id" v-text="label" />
```
Comment (JanAckermann) [inline src/components/OcDatepicker.vue:3]: I am not quite sure but we might need an value attribute for accessibility?
image

Comment (JammingBen) [reply src/components/OcDatepicker.vue:3]: Need to do some research on that 👀 

But I don't think it's required. The value here is basically what the user inputs. The same goes for the `text-input-fields` component, where we also don't have a value attribute. The calendar-modal itself may be another story, though...

Comment (JanAckermann) [reply src/components/OcDatepicker.vue:3]: Research done on my side -> not required

File: src/components/OcDatepicker.vue [hunk 3]
```diff
@@ -107,18 +118,48 @@ export default {
...
+      return `${this.id}-message`
+    },
+    additionalAttributes() {
+      const additionalAttrs = {}
+      if (this.descriptionMessage) {
+        additionalAttrs["aria-describedby"] = this.messageId
+      }
+      return { ...this.$attrs, ...additionalAttrs }
+    },
+    ariaInvalid() {
```
Comment (kulmann) [inline src/components/OcDatepicker.vue:154]: Not needed as of now. Please remove. :-) Same for the `aria-invalid` attribute in the template.

**Review by kulmann — CHANGES_REQUESTED** [review]

**Review by kulmann — APPROVED** [review]
👍

**Pull request merged**